.github/workflows/shellcheck.yml

```
@@ -34,8 +34,11 @@ jobs:
             github.com:443
             pkg-containers.githubusercontent.com:443
       - uses: actions/checkout@v2
-      - name: Install shellcheck
-        run: brew install shellcheck
+      - name: Force Install latest shellcheck
+        run: |
+          eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)" && \
+          brew install shellcheck && \
+          ln -s /home/linuxbrew/.linuxbrew/bin/shellcheck /usr/local/bin/shellcheck
```

can this be simplified to
```suggestion
          brew install shellcheck --prefix=/usr/local/bin/
```
?

---

hm, that didn't work - maybe setting the env var `HOMEBREW_PREFIX: /usr/local/bin/`?

---

I'll try this on my local later tonight, from my reading it seems most of these prefix related things are output only and not configuration options.